ui/src/themes/openems/scss/variables.scss

```diff
@@ -83,4 +83,40 @@ $font-family: var(--ion-font-family);
 
   /** font **/
   --ion-font-family: "Roboto", "Helvetica Neue", sans-serif;
+  --ion-background-color: #fff;
```

Comment:
Set an explicit background color to be able to use this variable with the energy monitor

Reply:
good